app/views/tracks/approaches/tooltip_locked.html.haml

```diff
@@ -1,10 +1,19 @@
-.c-generic-tooltip.text-14{ class: "!py-16 !px-24 !max-w-[420px]" }
-  %p.text-16.text-white.font-semibold.leading-170.mb-12
+.c-generic-tooltip.text-16.text-borderLight.leading-150{ class: "!py-16 !px-24 !max-w-[420px]" }
+  %p.text-lightPurple.font-semibold.mb-4
     Struggling with this exercise?
- 
-  %p.mb-4.leading-150
-    Unlock the Dig Deeper tab to see how the community approached and solved #{@exercise.title}. 
 
-  / TODO: add button to manually unlock help
-  .font-semibold.text-lightGold.text-14.leading-170.flex.flex-row.items-center
+  - if @solution&.downloaded? || @solution&.submitted?
+    %p.mb-12{ class: '!mt-0 ' }
+      Unlock the Dig Deeper tab to see how the community approached and solved #{@exercise.title}.
+
+    .flex.mb-10
+      .grow{ class: "w-[100%]" }
+        = button_to unlock_help_api_solution_url(@solution.uuid), method: :patch, class: 'btn-primary btn-s grow w-[100%] text-14 !py-4 !px-16' do
+          = graphical_icon 'unlock', css_class: "w-[14px] h-[14px]"
+          Unlock this tab
+  - else
+    %p.mb-12
+      You can only view Approaches for this exercise if you've downloaded or submitted a solution to this exercise.
+
+  .font-semibold.text-lightGold.text-14
```

Comment:
On the line below this:
1. Can you remove the word "Warning!"
2. Can you change "revealing Approaches" to "exploring the Dig Deeper tab"

Reply:
And apply this to the tooltip below too pls.